include/picongpu/param/species.param

```diff
@@ -43,7 +43,7 @@ namespace picongpu
  *  - particles::shapes::PCS : 3rd order
  *  - particles::shapes::P4S : 4th order
  *
- *  example:             using CICShape = particles::shapes::CIC;
+ *  example for using CIC shape: `using UsedParticleShape = particles::shapes::CIC;`
```

Comment:
I also think this is not necessary. We write literally prosa here, now you say:

"example using CIC shape using CIC shape"... I would have stayed with `example:     using UsedParticleShape = particles::shapes::CIC;`